micrometer-core/src/main/java/io/micrometer/core/instrument/logging/LoggingMeterRegistry.java

```diff
@@ -152,13 +152,33 @@ protected void publish() {
                                     loggingSink.accept(print.id() + " throughput=" + print.rate(count) +
                                             " mean=" + print.time(timer.mean(getBaseTimeUnit())));
                                 },
-                                meter -> loggingSink.accept(print.id() + StreamSupport.stream(meter.measure().spliterator(), false)
-                                        .map(ms -> ms.getStatistic().getTagValueRepresentation() + "=" + decimalOrNan(ms.getValue())))
+                                meter -> loggingSink.accept(writeMeter(meter, print))
                         );
                     });
         }
     }
 
+    String writeMeter(Meter meter, Printer print) {
+        return print.id() + StreamSupport.stream(meter.measure().spliterator(), false)
+                .map(ms -> {
+                    String msLine = ms.getStatistic().getTagValueRepresentation() + "=";
+                    switch (ms.getStatistic()) {
+                        case TOTAL:
+                        case MAX:
+                        case VALUE:
+                            return msLine + print.value(ms.getValue());
+                        case TOTAL_TIME:
+                        case DURATION:
+                            return msLine + print.time(ms.getValue());
+                        case COUNT:
+                            return msLine + print.rate(ms.getValue());
```

Comment:
This will turn out as `count=12/s` for example. Maybe it would be better as `throughput=` like other meter types? I can change this on merge so no pressing need to update just for this.